tests/replication/storage/test_events.py

```diff
@@ -90,7 +90,7 @@ def tearDown(self) -> None:
     def test_get_latest_event_ids_in_room(self) -> None:
         create = self.persist(type="m.room.create", key="", creator=USER_ID)
         self.replicate()
-        self.check("get_latest_event_ids_in_room", (ROOM_ID,), [create.event_id])
+        self.check("get_latest_event_ids_in_room", (ROOM_ID,), {create.event_id})
```

Comment:
I guess this works because if `s: Set` then `s == frozenset(s)`?

Reply:
```
>>> {1, 2, 3} == frozenset({1, 2, 3})
True
```

Reply:
You can compare equality of `frozenset` and `set`, yes. 👍